src/ovirtapi.js

```diff
@@ -34,6 +38,7 @@ OvirtApi = {
   },
   _httpGet ({ url, custHeaders = {} }: { url: string, custHeaders?: Object}): Promise<Object> {
     logDebug(`_httpGet start: url="${url}"`)
+    const requestId = notifyStart('GET', url)
```

Comment:
I would add integer global counter to make this requestId unique, the `url` is not enough.